sklearn/metrics/tests/test_classification.py

```diff
@@ -366,7 +366,8 @@ def test_matthews_corrcoef():
     y_true_inv = ["b" if i == "a" else "a" for i in y_true]
 
     assert_almost_equal(matthews_corrcoef(y_true, y_true_inv), -1)
-    y_true_inv2 = label_binarize(y_true, ["a", "b"]) * -1
```

Comment:
`y_true_inv2` is an array of 0 and 1 whereas `y_true` is an array of 'a' and 'b'. The test was passing just by chance as noted in https://github.com/scikit-learn/scikit-learn/pull/8094#issuecomment-268703047.